src/_palette.scss

```diff
@@ -53,8 +53,8 @@ $COCOA_500: #715344;
 $COCOA_700: #57392d;
 $COCOA_900: #3d211b;
 
-$SILVER_100: #fafafa;
-$SILVER_300: #d4d4d4;
+$SILVER_100: #fcfcfc;
+$SILVER_300: #e3e3e3;
```

Comment:
If you want to propose changes to the light style, please do that in a separate branch

Reply:
Will revert this.